src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const RENTAL_COST_PER_DAY = 40;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const SHORT_TERM_DISCUONT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const OVERALL_RENTAL_COST = RENTAL_COST_PER_DAY * days;
```

Comment:
It is customary to name static variables with a capital letter, if something is calculated in the variable, it is better to name it using the usual camelCase.